<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpesicficJobs</class>
 <widget class="QMainWindow" name="SpesicficJobs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vis utvalgte jobber</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>jobber.png</normaloff>jobber.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="labelInfo">
      <property name="text">
       <string>Her kan du gjøre et søk med spesifikke utvalg (dato, statusid, etc).</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelJobTitle">
      <property name="text">
       <string>Tittel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditJobTitle"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelCompanyName">
      <property name="text">
       <string>Bedrift</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditCompanyName"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelCityID">
      <property name="text">
       <string>Stedsnavn</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBoxCityName"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelStatusID">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBoxStatusName"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelDeadline">
      <property name="text">
       <string>Søknadsfrist</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditDeadline"/>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btnSearch">
      <property name="text">
       <string>Søk!</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QTableView" name="tableViewResults">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
